src/app/containers/Ad/Canonical/__snapshots__/index.test.jsx.snap

```diff
@@ -22,6 +22,7 @@ exports[`CanonicalAds Ads Snapshots should correctly render an Canonical leaderb
     <div>
       <div
         class="c0"
+        tabindex="-1"
```

Comment:
this is only needed on interactive elements such as iframes, links, buttons etc.

Reply:
Thank you for reviewing 👍 `tab-index` is no longer here, I was just trying out different ways to hide the ads. We will ask GNL to add `tab-index` directly to links/iframes as it is not possible to hide ads from tab order in Simorgh